plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java

```diff
@@ -93,7 +97,9 @@ public static ServiceManagerEnvironment loadServiceManagerEnvironment(IServer se
 	 */
 	public static ServiceManagerEnvironment loadServiceManagerEnvironment(IServer server) {
 		
-		String[] args = new String[]{CDKConstants.VAGRANT_CMD_SERVICE_MANAGER, CDKConstants.VAGRANT_CMD_SERVICE_MANAGER_ARG_ENV, "docker"};
```

Comment:
So you removed "docker" from the argument? But what about service manager 1.0.1 (CDK 2.0.0) support?